.claude/audit/scripts/diff.py

```
@@ -0,0 +1,110 @@
+    ap.add_argument("--census", default=DEFAULT_CENSUS)
+    ap.add_argument("--skill-dir", default=DEFAULT_SKILL_DIR)
+    ap.add_argument("--out", default=DEFAULT_OUT)
+    args = ap.parse_args()
+
+    data = json.load(open(args.census, encoding="utf-8"))
+    census, boiler = data["census"], set(data["boilerplate"])
+
+    texts = []
+    for ext in ("**/*.md", "**/*.cs"):
+        for p in glob.glob(os.path.join(args.skill_dir, ext), recursive=True):
+            texts.append(open(p, encoding="utf-8", errors="ignore").read())
```

Ruff (0.15.14)

[warning] Use `list.extend` to create a transformed list

(PERF401)